beancount/parser/parser_test.py

```diff
@@ -67,6 +69,19 @@ def test_parse_doc__noerrors(self, _, __, ___):
           Assets:US:Cash             -100 USD
         """
 
+    @unittest.skipIf(sys.platform != "win32", reason="test for win32 file separator")
```

Comment:
Why do you skip this test when executed on platforms other than Windows? As far as I know, the parser does not care if the path in the `document` directive is valid or not.

Reply:
Our grammer care
 https://github.com/beancount/beancount/blob/2a455c7987c8ae81bba761f5da3d74e4711d9e8e/beancount/parser/grammar.py#L773-L776

Reply:
Thanks. I didn't remember about that.